killerbee/dev_sewio.py

```diff
@@ -9,20 +9,20 @@
 #   http://www.sniffer.wislab.cz/sniffer-configuration/
 #
 
-import os
-import time
-import struct
-import time
+import os # type: ignore
+import time # type: ignore
+import struct # type: ignore
+import time # type: ignore
```

Comment:
ditto above as an example (i was trying not to comment on every single file where the same issue is)